test/functional/p2p_segwit.py

```diff
@@ -1149,25 +1203,24 @@ def test_standardness_v0(self, segwit_activated):
         self.utxo.append(UTXO(tx3.sha256, 0, tx3.vout[0].nValue))
         assert_equal(len(self.nodes[1].getrawmempool()), 0)
 
-
     # Verify that future segwit upgraded transactions are non-standard,
     # but valid in blocks. Can run this before and after segwit activation.
     def test_segwit_versions(self):
         self.log.info("Testing standardness/consensus for segwit versions (0-16)")
         assert(len(self.utxo))
-        NUM_TESTS = 17 # will test OP_0, OP1, ..., OP_16
-        if (len(self.utxo) < NUM_TESTS):
+        num_tests = 17  # will test OP_0, OP1, ..., OP_16
```

Comment:
in commit f7c7f8ecf3 [tests] p2p_segwit: Fix flake8 warnings

You change a constant to lowercase. Imo this makes the code harder to read, as mentioned in my previous review.

Reply:
sorry - recapitalized the ones that you mentioned in your review. Missed this one.